FundacionParaguaya/MentorApp
# Up to date screen

**Describe the changes**
This pull request implements the sync screens.

**To Test**
Specific steps to test when reviewing:

Dashboard:

The list on the Dashboard is limited to 5 items. Newest items appear on top. If there are no drafts, the list is not shown. These changes refer to issues:
- #299 
- #84 

Sync screen up to date:

When there are no drafts pending to sync but the user is online the sync screen shows Up to date component. This refers to issue:
- #278 

Sync screen offline:

When the user is offline, the offline screen is shown. If there are pending items, those show as well. This refers to:
- #279 

Sync screen sync in progress:

When the user has been offline and there are items pending and now the user is online, the sync in progress screen shows. User can see the progress of the sync and the items to be synced. This refers to:
- #295 

Translations:

Check translations on all of the sync screens.
